### ISSUE TITLE
Font size too small on 4k monitor on Linux

### ISSUE THREAD

**Issue by misev** [alex-hhh/ActivityLog2#52]
Screenshot to illustrate the issue:

![Screenshot_20201104_111441](https://user-images.githubusercontent.com/4786324/98098761-0cde8b80-1e8f-11eb-95f4-60ad906ed361.png)

I had the font sizes in the Summary boxes increased in the code, but have somehow managed to lose it with the latest git pull.

For the tab selector I increased it in the code with
```diff
diff --git a/rkt/widgets/tab-selector.rkt b/rkt/widgets/tab-selector.rkt
index 980853f..fbf0960 100644
--- a/rkt/widgets/tab-selector.rkt
+++ b/rkt/widgets/tab-selector.rkt
@@ -50,7 +50,7 @@
     (super-new)

     ;; Font used to draw the labels
-    (define font (send the-font-list find-or-create-font 9 'default 'normal 'normal
+    (define font (send the-font-list find-or-create-font 14 'default 'normal 'normal
                        #f 'smoothed))
```
The axis labels and legends in the graphs are similarly a bit small.

I think font size could be made dependent on screen resolution, but I'm not sure how difficult it would be.

**Comment by alex-hhh** [comment]
> The axis labels and legends in the graphs are similarly a bit small.

[plot-font-size](https://docs.racket-lang.org/plot/params.html#%28def._%28%28lib._plot%2Fmain..rkt%29._plot-font-size%29%29) will set the font size used for labels (there are also parameters for the font family and other things.   I am not sure where best to place it, but this will work for a quick fix:

```diff
diff --git a/rkt/main.rkt b/rkt/main.rkt
index 6215e0e..e7dadf8 100644
--- a/rkt/main.rkt
+++ b/rkt/main.rkt
@@ -19,11 +19,13 @@
          "dbutil.rkt"
          "dialogs/first-run.rkt"
          "toplevel.rkt"
-         "utilities.rkt")
+         "utilities.rkt"
+         (only-in plot plot-font-size))
 
 (provide main)
 
 (define dbfile-key 'activity-log:database-file)
+(plot-font-size 24)
 
 (define (main)
   (dbglog "main started, version ~a, ~a, ~a"
```

This however will not update some of the hover labels that show up on plots, for those, some changes in dependent packages are needed.

----

> I think font size could be made dependent on screen resolution

This is the bit that I am confused about.  On Windows, I have a 2K monitor (1.5 scale) and a Full HD one (1K?).  The fonts are automatically scaled as needed.  For example, I can start the application on the 2K screen and, if I move it to the 1K screen the fonts are re-scaled so that the text is approximately the same size.  The scaling happens at runtime by simply dragging the window between the two monitors.  You can even see when the scaling happens, as if the window is more than 50% on the 2K screen, the part that is on the Full HD screen is actually bigger.

**Comment by misev** [comment]
Yes in general the scaling works well, only some widgets seem to have trouble:
- tab selector
- Session overview boxes (which I'm not very concerned about as the same information is on the Activities tab)
- plot labels, which got very well fixed with your suggestion above
